### PR TITLE
[nri-statsd] Add support for the nri-statsd integration

### PR DESCRIPTION
**Pull request by bambash**
#### Is this a new chart
Yes

#### What this PR does / why we need it:
This PR adds a chart for the nri-statsd integration.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
The statsd integration doesn't require a `licenseKey`, rather it needs an `insightsKey`. I saw that the global variable already existed for `insightsKey` but wasn't referenced in any other charts. I've updated the documentation and `nri-bundle` chart to include `nri-statsd` and the reference to `global.insightsKey`. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by arvdias — APPROVED** [review]

**Comment by arvdias** [comment]
@bambash Thank you for the contribution! Can you just please check the conflicted file?

**Comment by bambash** [comment]
@arvdias done - thank you!

**Comment by arvdias** [comment]
Thank you @bambash

**Review by arvdias — APPROVED** [review]

**Comment by arvdias** [comment]
@roobre @gsanchezgavier @paologallinaharbur I'm not currently set in the owners file, can you review this please?

File: charts/nri-bundle/README.md [hunk 1]
```diff
@@ -85,19 +85,23 @@ Then, to install this chart, run the following command:
...
 ```
 
 [1]: https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key
-[2]: https://github.com/helm/charts/tree/master/stable/kube-state-metrics
-[3]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure
-[4]: https://github.com/newrelic/helm-charts/tree/master/charts/nri-prometheus
-[5]: https://github.com/newrelic/helm-charts/tree/master/charts/nri-metadata-injection
-[6]: https://github.com/newrelic/helm-charts/tree/master/charts/nri-kube-events
-[7]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging
+[2]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#insights-insert-key
```
Comment (paologallinaharbur) [inline charts/nri-bundle/README.md:100]: Are these links referenced somewhere? 
It seems something is missing at the beginning of the readme

File: charts/nri-bundle/requirements.lock [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
 - name: nri-kube-events
   repository: file://../nri-kube-events
-  version: 1.3.4
+  version: 1.4.0
 - name: newrelic-logging
   repository: file://../newrelic-logging
   version: 1.4.2
-digest: sha256:e3debc8b192fe2ee26342de591855d4ff9e80f6253bb7710861ec2315e8e5d1a
-generated: "2020-11-26T12:46:14.62338+01:00"
+- name: nri-statsd
```
Comment (paologallinaharbur) [inline charts/nri-bundle/requirements.lock:20]: Can we split the PR of the new chart from the update of nri-bundle?

**Comment by bambash** [comment]
@paologallinaharbur thanks for reviewing - I removed the nri-statsd chart from nri-bundle and its README

File: charts/nri-bundle/requirements.lock (hunk 2 above)
Comment (bambash) [reply charts/nri-bundle/requirements.lock]: @paologallinaharbur 👍 definitely, that sounds good. I stripped out all of the new chart references from this PR. Once this is merged I can open another to update nri-bundle. Does that sound alright?

**Review by roobre — APPROVED** [review]
Approving on behalf of @arvdias

**Comment by roobre** [comment]
@bambash Please rebase `master` whenever you have time :)

**Comment by bambash** [comment]
@roobre done! thank you

**Comment by roobre** [comment]
Hi,

It seems that more conflicts have been introduced. Likely this was because an upgrade on the `nri-bundle` that had high priority :(

Could you rebase again, @bambash? Thanks!

**Comment by roobre** [comment]
lgtm, thanks for the effort!

**Pull request merged**